src/main/java/seedu/address/logic/parser/UnmarkCommandParser.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.parser;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.MarkCommand;
+import seedu.address.logic.commands.UnmarkCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+/**
+ * Parses input arguments and creates a new MarkCommand object
+ */
+public class UnmarkCommandParser implements Parser<UnmarkCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the MarkCommand
+     * and returns a MarkCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public UnmarkCommand parse(String args) throws ParseException {
+        try {
+            Index index = ParserUtil.parseIndex(args);
+            return new UnmarkCommand(index);
+        } catch (ParseException pe) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, MarkCommand.MESSAGE_USAGE), pe);
+        }
+    }
+
+}
```

Comment:
Add newline here as well!

Reply:
Got it, will add in the newlines in a new commit!